doc/CompilationPython.md

```diff
@@ -0,0 +1,189 @@
+| [Tutorials Home](index.md) | [Previous](ImportExport.md) | [Next](PythonModule.md) |
+| :--- | :---: | ---: |
+
+# Compiling libpointmatcher with python
+
+This tutorial presents the different steps of compiling pypointmatcher, the libpointmatcher's python module, on Ubuntu and Mac OS X.
+
+## Prerequisites
+
+To get started, you will obviously need the same prerequisites as libpointmatcher, but also some additional dependencies as listed here :
+
+| Name | Version <br> (Tested August 2020 on Ubuntu 18.04) |
+| :--- | :---: |
+| pybind11 | 2.5.0 |
+| Python3 | 3.6.9 |
+|||
+| **Dependencies** | |
+| catch | 1.10.0 |
+| python3-dev | 3.6.7 |
+| pytest | 5.4.3 |
+
+`catch` and `python3-dev` need to be installed with the package manager :
+
+```bash
+sudo apt install catch python3-dev
+```
+
+and `pytest` with `pip` :
+
+```bash
+pip3 install pytest
+```
+
+> ***Note :*** pybind11 is also compatible with Python2.7, which means that it is also possible to generate a module for Python2.7. But, this hasn't been tested. You can refer to the pybind11 [official documentation](https://pybind11.readthedocs.io/en/latest/index.html) for more informations on what you can do with pybind11 and how to do it.
```

Comment:
Python 2.7 is also EOL: https://www.python.org/doc/sunset-python-2/

Reply:
I knew, but I put it there just in case some people would like to know that fact. I have no problem to remove it, if you think that it is irrelevant.